src/platform/forms-system/src/js/web-component-patterns/titlePattern.js

```diff
@@ -2,14 +2,42 @@
 import React from 'react';
 import { isReactComponent } from '~/platform/utilities/ui';
 
-export const Title = ({ title, description, headerLevel = 3, classNames }) => {
+export const Title = ({
+  title,
+  description,
+  headerLevel = 3,
+  headerStyleLevel,
+  classNames,
+}) => {
   const CustomHeader = `h${headerLevel}`;
-  const color = headerLevel === 3 ? 'gray-dark' : 'black';
-  const className = classNames || `vads-u-color--${color} vads-u-margin-top--0`;
+  const style = headerStyleLevel
```

Comment:
I know this is what we were looking at earlier today but this will result in a header level being potentially different from the header style level, right? ie a `<h1>` could have an `h3` style or something? Is there a more expected behavior we could go for here? (ie, changing `<h1>`s to `<h3>`s if the style calls for `h3`)

Feel free to ignore if I'm totally off base here! 😄

Reply:
yeah this is the best way to go about this I think, the reason you'd want semantic h1, but style h3, is because screen readers care about semantics like a proper hierarchy where it should always start with h1, then h2s, then h3s etc... but we may want arbitrary styling regardless of the semantics